lib/entities/app-bundle.ts

```diff
@@ -16,6 +16,15 @@ interface ActionManifestProps {
   description: string
   category: string
   path: string
+  allowNetworks?: string[]
+}
+
+interface DeliveryFunctionManifestProps {
+  id: string
+  name: string
+  description: string
+  path: string
+  allowNetworks?: string[]
 }
```

Comment:
Also `entryFile` or no? Just from looking at [this guy](https://github.com/contentful/apps/blob/master/examples/hosted-delivery-function-templates/typescript/contentful-app-manifest.json)

Reply:
`entryFile` is for building code to upload. It's not something our API cares about